RNTester/e2e/__tests__/Switch-test.js

```diff
@@ -13,69 +13,85 @@
 const jestExpect = require('expect');
 
 describe('Switch', () => {
-  beforeEach(async () => {
+  beforeAll(async () => {
```

Comment:
Switched this to speed up the test so we reload once for the file instead of after every test

Reply:
This makes sense; I switched to `beforeEach` to try to decrease flake of the Switch tests, but it wasn't 100% successful, and it makes sense that the filter might serve the same purpose.